plyer/platforms/linux/inappbrowser.py

```diff
@@ -0,0 +1,14 @@
+
+from plyer.facades.inappbrowser import InAppBrowser
+import webbrowser
+
+
+class LinuxAppBrowser():
+
+    def _open_url(self, **kwargs):
+        url = kwargs.get('url', '')
```

Comment:
Using the right protocol is enough on desktops, but `webbrowser.open('http://')` for some reason kills Firefox on Android. Funny.